tools/applets/dds_channel.py

```diff
@@ -164,7 +163,7 @@ def initialize_gui(self):
         self.freq_box.setSingleStep(0.1)
         self.freq_box.setFont(spinboxfont)
         self.freq_box.setKeyboardTracking(False)
-        MHz_to_Hz = 1.e6
+        MHz_to_Hz = 1.0e6
```

Comment:
not sure what we usually do for this but would it be easier to define it as a global or class constant? It seems we use it 3+ times

Reply:
I agree. This is out of the scope of this PR though.